Tribler/community/chant/orm.py

```diff
@@ -0,0 +1,53 @@
+
+from datetime import datetime
+from pony import orm
+from Tribler.community.chant.MDPackXDR import serialize_metadata_gossip
+
+db = orm.Database()
+
+class Peer(db.Entity):
```

Comment:
This name conflicts with ipv8's Peer class.

Reply:
Oops...